sources/packages/backend/workflow/src/workflow-definitions/load-assessment-consolidated-data.bpmn

```diff
@@ -129,6 +129,7 @@
           <zeebe:header key="partner1HasFedralProvincialPDReceipt" value="$.supportingUsers.Partner1.supportingData.hasFedralProvincialPDReceipt" />
           <zeebe:header key="partner1HasTotalIncomeAssistance" value="$.supportingUsers.Partner1.supportingData.hasTotalIncomeAssistance" />
           <zeebe:header key="studentDataExceptionalExpenseAmount" value="$.data.exceptionalExpenseAmount" />
+          <zeebe:header key="studentDataParents" value="$.data.parents" />
```

Comment:
Similar to dependants we just map the property `ableToReport` right? excluding the `fullName`

Reply:
Removing this file from now to have it added when the form.io change is also added. Hope that is ok.